apps/prairielearn/python/prairielearn.py

```diff
@@ -1090,7 +1101,22 @@ def string_to_number(s, *, allow_complex=True):
         return None
 
 
-def string_fraction_to_number(a_sub, allow_fractions=True, allow_complex=True):  # noqa: FBT002 (downstream callers already use this signature)
+class _PartialDataFormatError(TypedDict):
```

Comment:
Nit, for consistency with the other type:

```suggestion
class _PartialDataFormatErrors(TypedDict):
```

Reply:
Changed in b81eb29eae1c5f6917959f7237f2b63e981d21f1.